tests/longhaul/base_block_test.go

```diff
@@ -22,7 +22,7 @@ func setUpRookAndPoolInNamespace(t func() *testing.T, namespace string, storageC
 
 	i := installer.NewK8sRookhelper(kh.Clientset, t)
 	if !kh.IsRookInstalled(namespace) {
-		isRookInstalled, err := i.InstallRookOnK8sWithHostPath(namespace, "bluestore", "/temp/rookBackup")
+		isRookInstalled, err := i.InstallRookOnK8sWithHostPathAndDevices(namespace, "bluestore", "/temp/rookBackup", true, 3)
```

Comment:
how does the dataDirHostPath get cleaned up?

Reply:
the long haul tests will have to worry about that, I believe new machines are provisioned for each longhaul. @dangula?

Reply:
yes, only long haul tests use  the dataDirHostPath, and we provision new machines for each run.